src/main.lua

```diff
@@ -2015,27 +2012,6 @@ addEventingHandler(ActionMap.Epoch, utils.hasMatchingTag("Action", ActionMap.Epo
 	Send(msg, { Target = msg.From, Action = "Epoch-Notice", Data = json.encode(epoch) })
 end)
 
-addEventingHandler(ActionMap.Epochs, utils.hasMatchingTag("Action", ActionMap.Epochs), function(msg)
-	local allEpochs = epochs.getEpochs()
-
-	-- the json encoder will error on a table with numeric keys that don't start at 1.
-	-- at genesis, epochs will start indexing at 0 and within 2 epochs will continue indexing
-	-- from 2 (and so on as epochs continue to be pruned), so work around that by sorting the
-	-- epoch index keys and inserting the epochs in order into a fresh array
-	local keys = {}
-	for key in pairs(allEpochs) do
-		table.insert(keys, key)
-	end
-	table.sort(keys)
-
-	local sortedEpochs = {}
-	for _, key in ipairs(keys) do
-		table.insert(sortedEpochs, allEpochs[key])
-	end
-
-	Send(msg, { Target = msg.From, Action = "Epochs-Notice", Data = json.encode(sortedEpochs) })
-end)
```

Comment:
removing this handler - it does not scale well and is not needed by anything downstream